src/openlifu/db/database.py

```diff
@@ -377,22 +392,11 @@ def load_subject(self, subject_id, options=None):
         self.logger.info(f"Loaded subject {subject_id}")
         return subject
 
-    def load_volume(self, subject, volume_id):
-        raise NotImplementedError("Volume is not yet implemented")
-        volume_filename = self.get_volume_filename(subject.id, volume_id)
+    def load_volume(self, subject_id, volume_id):
+        volume_filename = self.get_volume_filepath(subject_id, volume_id)
         volume = Volume.from_file(volume_filename)
         return volume
```

Comment:
It's not enough to just return the volume metadata, we need to also get the absolute file path of the volume file (see https://github.com/OpenwaterHealth/OpenLIFU-python/issues/91#issuecomment-2365232909 ). The user needs to know where the volume file lives in order to load it themselves.

The way to get the path to the volume file is to use the path to the volume metadata json, take the parent directory of that, and then use the `filename` that should also be stored in the volume info json